prisma/schema.prisma

```diff
@@ -9,6 +9,7 @@ datasource db {
 
 model Comment {
   id          Int       @id @default(autoincrement())
+  accidentId  Int       @db.Integer
```

Comment:
We use `gibdd_slug` in Django, which is of type string: https://github.com/dtpstat/dtp-stat/search?q=gibdd_slug (internal link). I don’t mind going for integer because all slugs seem to be integer-like. However, we might need to change that later.

Reply:
Updated to string